test/test_rocrand_kernel_sobol64.cpp

```diff
@@ -391,13 +481,28 @@ TEST_P(rocrand_kernel_sobol64_poisson, rocrand_poisson)
     HIP_CHECK(hipMalloc(&output, output_size * sizeof(unsigned int)));
     HIP_CHECK(hipDeviceSynchronize());
 
+    hipStream_t stream = 0;
+    hipGraph_t graph;
+    hipGraphExec_t graph_instance;
+    if (use_graphs)
+    {
+        // Default stream does not support hipGraph stream capture, so create a non-blocking one
+        HIP_CHECK(hipStreamCreateWithFlags(&stream, hipStreamNonBlocking));
+        graph = test_utils::createGraphHelper(stream);
+    }
+
     hipLaunchKernelGGL(
         HIP_KERNEL_NAME(rocrand_poisson_kernel<state_type>),
         dim3(8), dim3(32), 0, 0,
         output, m_vector, output_size, lambda
     );
     HIP_CHECK(hipGetLastError());
```

Comment:
Idem.
```suggestion
    hipLaunchKernelGGL(
        HIP_KERNEL_NAME(rocrand_poisson_kernel<state_type>),
        dim3(8), dim3(32), 0, stream,
        output, m_vector, output_size, lambda
    );
    HIP_CHECK(hipGetLastError());
```

Reply:
Thanks. As above - I'll remove all hipGraph tests from the "kernel test" files.